Gem/Settings/ViewModels/CurrencySceneViewModel.swift

```diff
@@ -66,6 +70,70 @@ class CurrencySceneViewModel: ObservableObject {
 
 extension Locale.Currency {
     var title: String {
-        return String(format: "%@ - %@", identifier, Locale.current.localizedString(forCurrencyCode: identifier) ?? .empty)
+        let localizedCurrencyId = Locale.current.localizedString(forCurrencyCode: identifier) ?? .empty
```

Comment:
can you add unit test for this one? to make sure we properly present the title

Reply:
will add